server/config/config.go

```diff
@@ -1184,6 +1184,10 @@ type StoreLabel struct {
 	Value string `toml:"value" json:"value"`
 }
 
+// RejectLeader is the label property type that suggests a store should not
+// have any region leaders.
+const RejectLeader = "reject-leader"
```

Comment:
how about `server/core/store.go`?

Reply:
I think it can be regarded as a kind of config to some degree.